### ISSUE TITLE
Bug : FXCM Server reports an error: Invalid instrument_id..

### ISSUE THREAD

**Issue by JamesKBowler** [fxcm/fxcmpy#4]
Hello,
Any ideas on the below errors?

Thanks

James

**Errors**
```
TRY/JPY |ERROR|2018-06-22 15:13:31,680|Server reports an error: {'error': 'Invalid instrument_id.', 'executed': False}.
FXCM Server reports an error: Invalid instrument_id..

USD/CNH |ERROR|2018-06-22 15:13:35,199|Server reports an error: {'error': 'Invalid instrument_id.', 'executed': False}.
FXCM Server reports an error: Invalid instrument_id..

CHN50 |ERROR|2018-06-22 15:13:39,295|Server reports an error: {'error': 'Invalid instrument_id.', 'executed': False}.
FXCM Server reports an error: Invalid instrument_id..


```
**Test Script**

```python
import fxcmpy
TOKEN = ""
con = fxcmpy.fxcmpy(access_token=TOKEN, log_level='error')
timeframe = 'm1'
dt_to = datetime.utcnow()
dt_fm = dt_to - timedelta(minutes=5)
for i in con.get_offers('list'):
    asset_symbol = i['currency']
    try:
        con.get_candles(asset_symbol, period=timeframe, start=dt_fm, end=dt_to)
    except Exception as e:
        print(asset_symbol, e)
```

**Comment by FXCMAPI** [comment]
Hello James

We are working on it, 
temp solution is please subscribe those pairs on out tsweb 2.0 GUI 
https://tradingstation.fxcm.com/

best regards
Lei

**Comment by JamesKBowler** [comment]
Hi Lei,
Thank you for the speedy response.
Not sure I follow, they are already subscribed, do you mean unsubscribe?

Regards

James

**Comment by FXCMAPI** [comment]
Hello James
the function get_candles that facmpy wrapper used was binded to tsweb GUI. 
if those currencies are not subscribed on GUI, you will get those error.
we are working on it to change that.
if you still have problem, please provide login and token,

thanks
Lei

**Comment by FXCMAPI** [comment]
This has been fixed in latest version 1.1.28.
closing this case.
